.circleci/config.yml

```diff
@@ -155,12 +155,14 @@ jobs:
 
 workflows:
   version: 2
-  build_test_27_publish:
+  build_test_publish:
     jobs:
-      - build_python27
+      - build_python36
+      - build_python38
       - publish_github:
           requires:
-            - build_python27
+            - build_python38
+            - build_python36
```

Comment:
Any particular reason this ordering is reversed?

Reply:
Yes - I added 3.8 first, then 3.6 :)

Reply:
08e76b2